examples/simple_project/main.tf

```diff
@@ -28,8 +28,10 @@ locals {
   subnet_03 = "${var.network_name}-subnet-03"
 }
 
+# [START customvpc_create]
 module "test-vpc-module" {
-  source       = "../../"
+  source       = "terraform-google-modules/cloud-router/google"
```

Comment:
The source and version here should be based on this module, not Cloud Router.

Reply:
Got it